Tests/DataDefinitionTests.cs

```diff
@@ -186,5 +189,21 @@ public void TestAddOrUpdateBodyUpdate()
             Assert.AreEqual("testSystem 1", actualBody.bodyname);
             Assert.AreEqual(moon, actualBody.bodyType);
         }
+
+        [TestMethod]
+        [DeploymentItem("vehicle.json")]
```

Comment:
This may not be 100% compatible with my code coverage branch (which seems to fail whenever presented with a test with a deployment item). Consequently, the code coverage branch embeds the resources into Tests.dll where code coverage can be happily evaluated. 
TLDR: Some post-merge patching may be required if both PRs are accepted. 🤷‍♂